src/update_cfgs.rs

```diff
@@ -0,0 +1,85 @@
+use log::warn;
+
+pub fn get_cfg_condition(object_name: &str, cfg_condition: &Option<String>) -> Option<String> {
```

Comment:
I'm surprised clippy does not warn about that. You usually want `Option<&str>`

Reply:
(for the parameter)

Reply:
Well, in this case it mostly depends on what we have. And we do have `Option<String>`, so I think it's better to pass a reference to t rather than calling `.as_deref()` on every instance.